vm/Cargo.toml

```diff
@@ -4,12 +4,6 @@ version = "0.1.0"
 authors = ["Steven Stanfield <stanfield@scarecrowtech.com>"]
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[features]
```

Comment:
I guess these are not used anymore?  I forget what gc was for since you kinda need it...  nohelmet was allow the use of more sketchy (but maybe faster) unsafe code.  Probably good if that one is gone.

Reply:
I came to the conclusion that they were entirely unused because I couldn't find any usage like an annotation `#[cfg(feature = "gc")]`. I agree you probably started with the garbage collection being optional and since made it mandatory. So no objection to removing these?